### ISSUE TITLE
Docs refer to istio-citadel-plugin-certs.yaml but it doesn't exist

### ISSUE THREAD

**Issue by spikecurtis** [istio/istio.github.io#1836]
https://preliminary.istio.io/docs/tasks/security/plugin-ca-cert.html says to run the following command

```
kubectl apply -f install/kubernetes/istio-citadel-plugin-certs.yaml
```

but the file does not exist in the release zip, nor does it appear in the istio.io/istio source tree, nor is it created by `updateVersion.sh`.  I'm not sure how to get this file!

**Comment by liminw** [comment]
According to @lei-tang, this was due to PR https://github.com/istio/istio/pull/6585, where all the templates are removed, but the corresponding helm charts are not created. 

A similar problem happened for health check (https://github.com/istio/istio/issues/6922). 

Two options:
1. Revert PR https://github.com/istio/istio/pull/6585. We should remove templates only after all the helm charts are created to completely replace the templates. 
2. Identify the missing files one by one, and fix them accordingly. I have no idea how many missing files are there due to this.  

/cc @ymesika @wenchenglu @wattli @myidpt

**Comment by liminw** [comment]
I plan to revert this PR istio/istio#6585, because it caused several issues of missing yaml files. @ymesika, let me know if you have any concern.

**Comment by frankbu** [comment]
Don't revert https://github.com/istio/istio/pull/6585. Fix the problems one by one. The instructions need to be changed to use helm.

There is already a PR that fixes this specific issue: https://github.com/istio/istio.github.io/pull/1757

**Comment by wattli** [comment]
@frankbu , i don't think it is a good idea to simply remove all files that are still actively listed in our web page, and wait for people to find them out one by one, and fix them one by one. For this issue it's one file, but that PR deleted 20+ of them, which should not happen.

I suggest to revert #6585 and update the reference to every single of them before we actually delete it. 

@rshriram @geeknoid @louiscryan @smawson

**Comment by ymesika** [comment]
@wattli What 20+ files? The output of the old `updateVersion.sh` generated 13 yamls (+ addons)
Not generated Citadel stuff by the new one are:
- `istio-citadel-plugin-certs.yaml`
- `istio-citadel-standalone.yaml`
- `istio-citadel-with-health-check.yaml`

The last one has been covered by https://github.com/istio/istio/pull/6961 which raise a question of @rshriram, which I tend to agree with, of why we are not always enabling it and getting rid of the flag + doc. It's still waiting for your comment.

So all in all we are left with a couple of Citadel files not generated. So @frankbu one by one approach makes sense. 

To be honest those YAMLs can be embedded into the doc for a copy-paste instructions rather than pretty static templates.

**Comment by ymesika** [comment]
I think the easiest solution is to just add back those 3 missing files to the release: https://github.com/istio/istio/pull/7178

**Comment by frankbu** [comment]
@wattli I'm not suggesting we wait around for users to find them one by one, I'm suggesting that whoever is responsible for these tasks (who wrote them?) should fix them one by one asap.

**Comment by sdake** [comment]
Re the citadel standalone helm PR here: https://github.com/istio/istio/pull/5292

This needs significant work and will not be ready for 1.0 - so suggest following @frankbu 's guidance.

Cheers
-steve

**Comment by frankbu** [comment]
fixed by #1757

**Comment by wattli** [comment]
@frankbu, my point is we should not make breaking change especially at this pre-1.0 moment.
